modules/system/assets/ui/js/input.trigger.js

```diff
@@ -140,6 +140,19 @@
                 .trigger('empty.oc.triggerapi')
                 .trigger('change')
         }
+        else if (action == 'fill' && status) {
+            this.$el
+                .not('input[type=checkbox], input[type=radio], input[type=button], input[type=submit]')
+                .val('')
```

Comment:
What is this doing? Looks like copy paste from the empty action

Reply:
@JLyons009 could you merge both actions (empty & fill) in one and just with ternarri operator use it for action|(true/false checkbox)

Reply:
Done.  Let me know if any more revisions are needed.  Thanks!